src/app/pages/Send/hooks/use-evm-estimation-data.ts

```diff
@@ -31,6 +34,9 @@ export const useEvmEstimationData = (
   toFilled?: boolean,
   amount?: string
 ) => {
+  const tokenMetadata = useEvmTokenMetadataSelector(network.chainId, assetSlug);
+  const collectibleMetadata = useEvmCollectibleMetadataSelector(network.chainId, assetSlug);
```

Comment:
```suggestion
  export const useEvmAssetMetadata = (
  chainId: number,
  assetSlug: string
): EvmTokenMetadata | EvmCollectibleMetadata | undefined => {
  const tokenMetadata = useEvmTokenMetadataSelector(chainId, assetSlug);
  const collectibleMetadata = useEvmCollectibleMetadataSelector(chainId, assetSlug);

  return tokenMetadata ?? collectibleMetadata;
};
```

use in all places like that